frontend/src/components/VPopover/meta/VPopover.stories.mdx

```diff
@@ -64,23 +60,23 @@ const ControlStory = (args, { argTypes }) => ({
       </template>
     </VPopover>
   `,
+  components: { VPopover, VButton },
+  setup() {
+    return { args }
+  },
 })
 
-export const Control = ControlStory.bind({})
-Control.args = {}
-
-const TwoPopoverStory = (args, { argTypes }) => ({
-  props: Object.keys(argTypes),
+export const TwoPopoverStory = (args) => ({
```

Comment:
`TwoPopoverStory` and `SinglePopoverStory` can be merged using a prop and `v-for` to generate as many popovers as needed. But that's an improvement, not a blocker.